server/python/inventory.py

```diff
@@ -2,61 +2,49 @@
 inventory.py
 Stripe Payments Demo. Created by Adrienne Dreyfus (@adrind).
 
-Simple library to store and interact with orders and products.
-These methods are using the Stripe Orders API, but we tried to abstract them
-from the main code if you'd like to use your own order management system instead.
+Simple library to store and interact with products and SKUs.
+These methods are using the Stripe Product API, but we tried to abstract them
+from the main code if you'd like to use your own product and order management system instead.
 """
 
 import stripe
 import os
 from functools import reduce
-from stripe_types import Order, Product
+from stripe_types import Product
 from dotenv import load_dotenv, find_dotenv
 
 load_dotenv(find_dotenv())
 
 stripe.api_key = os.getenv('STRIPE_SECRET_KEY')
-stripe.api_version = '2018-02-06'
+stripe.api_version = '2019-02-11'
 
 
 class Inventory:
     @staticmethod
-    def create_order(currency: str, items: list, email: str, shipping: dict, create_intent: bool) -> Order:
-        order = stripe.Order.create(currency=currency, items=items,
-                                    email=email, shipping=shipping, metadata={'status': 'created'})
-        if create_intent:
-            # Create PaymentIntent to represent customers intent to pay this order.
-            # Note: PaymentIntents currently only support card sources to enable dynamic authentication:
-            # https://stripe.com/docs/payments/dynamic-authentication
-            payment_intent = stripe.PaymentIntent.create(
-                amount=order['amount'], currency=currency, metadata={'order': order['id']}, allowed_source_types=['card'])
-            # Add PaymentIntent to order object so our frontend can access the client_secret.
-            # The client_secret is used on the frontend to confirm the PaymentIntent and create a payment.
-            # Therefore, do not log, store, or append the client_secret to a URL.
-            order['paymentIntent'] = payment_intent
-        return order
-
-    @staticmethod
-    def retrieve_order(order_id: str) -> Order:
-        return stripe.Order.retrieve(order_id)
+    def calculate_payment_amount(items: list) -> int:
+        product_list = stripe.Product.list(
+            limit=3, stripe_version='2018-02-28')
```

Comment:
Can we make a comment here explaining why we're using the old version?

Reply:
We can use the same comment you use in the node app! 
```
# For product retrieval and listing set API version to 2018-02-28 so that skus are returned.
```

Reply:
Fixed this already!